syft/pkg/license.go

```diff
@@ -68,11 +70,15 @@ func NewLicense(value string) License {
 		Value:          value,
```

Comment:
should there be empty checks for this value so we don't create licenses with "" as a value?